package.json

```diff
@@ -17,12 +17,13 @@
   },
   "homepage": "https://github.com/lblod/delta-consumer#readme",
   "dependencies": {
-    "@lblod/mu-auth-sudo": "0.6.1",
+    "@lblod/mu-auth-sudo": "0.6.2",
```

Comment:
consider bumping the template to 1.9.1 and just using auth-sudo from the template